fury/material.py

```diff
@@ -60,22 +60,30 @@ def manifest_standard(actor, ambient_level=0, ambient_color=(1, 1, 1),
         fragment level.
 
     """
-    prop = actor.GetProperty()
+    try:
```

Comment:
This was added because `contour_from_label` was throwing an exception when trying to access the `Property` object. See `test_material` and `test_manifest_standard`.

Reply:
ok, I will look into it